addons/project/models/project.py

```diff
@@ -1631,6 +1631,8 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
         fields_list = ([f.split(':')[0] for f in fields] or [])
         if groupby:
             fields_list += [groupby] if isinstance(groupby, str) else groupby
+            # only take field name when having ':' e.g 'date_deadline:week' => 'date_deadline'
+            fields_list = [f.split(':')[0] for f in fields_list]
```

Comment:
So now, you will do `.split(':')[0]` twice for the field names contained in the `fields` parameter... 😅

You could just do that for the content in `groupby` parameter.
```suggestion
            fields_groupby = [groupby] if isinstance(groupby, str) else groupby
            # only take field name when having ':' e.g 'date_deadline:week' => 'date_deadline'
            fields_list += [f.split(':')[0] for f in fields_groupby]
```

Or you could correctly the suggestion from @nle-odoo